lib/v2/weibo/group.js

```diff
@@ -0,0 +1,95 @@
+const querystring = require('querystring');
+const got = require('@/utils/got');
+const config = require('@/config').value;
+const weiboUtils = require('./utils');
+const { fallback, queryToBoolean } = require('@/utils/readable-social');
+
+module.exports = async (ctx) => {
```

Comment:
https://github.com/DIYgod/RSSHub/blob/9bdba060c7962d734745c8d6b9264da9a532b51d/lib/routes/disqus/posts.js#L5-L7
Similarly, you can throw an error if the cookie is not set